chat-client/src/client/mynahUi.ts

```diff
@@ -30,10 +30,38 @@ export interface InboundChatApi {
     showError(params: ErrorParams): void
 }
 
-export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [MynahUI, InboundChatApi] => {
+// Exposing MynahUiHelpers to make the code easier to test
+export interface MynahUiHelpers {
+    handleChatPrompt: (mynahUi: MynahUI, tabId: string, prompt: ChatPrompt, eventId?: string) => void
+}
```

Comment:
Will there be other things that needs access to this helper? Looks like the helper only needs to access `messenger` in the `createMynahUi` scope, could it just take an additional argument and be moved outside of the function?